### PR TITLE
Skeleton

### PR DESCRIPTION
**Pull request by BBriset**
Importing modules, creating pages and reusable components skeleton

### CODE REVIEW

**Review by HeloiseFerla — DISMISSED** [review]
Ok for me !

File: src/components/UserProfil/style.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+const SUserProfil = styled.main``;
```
Comment (Hacene010) [inline src/components/UserProfil/style.jsx]: In english, you must add a 'e' to profile : )

**Review by Hacene010 — DISMISSED** [review]
It's ok but take a look at my comment

**Review by HeloiseFerla — APPROVED** [review]
Ok !

**Review by Hacene010 — APPROVED** [review]
All good

**Pull request merged**